### PR TITLE
add debian 11 to ci tests

### PR DESCRIPTION
**Pull request by bryceml**
### What does this PR do?
Adds debian 11 (not released yet) to CI pipelines so we can easily release when debian 11 actually comes out.

### CODE REVIEW

File: cicd/jenkins/pr-debian-11-amd64-py3-pytest [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    distro_name: 'debian',
+    distro_version: '11',
+    distro_arch: 'amd64',
+    env: env,
+    golden_images_branch: 'master',
+    jenkins_slave_label: 'kitchen-slave',
+    nox_env_name: 'pytest-zeromq',
+    nox_passthrough_opts: '--ssh-tests',
+    python_version: 'py3',
+    //splits: ['unit', 'integration', 'multimaster'],
```
Comment (s0undt3ch) [inline cicd/jenkins/pr-debian-11-amd64-py3-pytest:15]: This comment can go away

File: cicd/jenkins/pr-debian-11-amd64-py3-pytest [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    testrun_timeout: 6,
+    use_spot_instances: true,
+    //fast_slow_staged_testrun: true
```
Comment (s0undt3ch) [inline cicd/jenkins/pr-debian-11-amd64-py3-pytest:18]: This one too

**Review by s0undt3ch — CHANGES_REQUESTED** [review]

**Comment by bryceml** [comment]
re-run full centos-7

**Comment by bryceml** [comment]
re-run full arch

**Comment by bryceml** [comment]
re-run full mojave

**Comment by bryceml** [comment]
re-run full windows

**Review by krionbsd — APPROVED** [review]

**Review by joechainz — APPROVED** [review]
Looks Good

**Review by s0undt3ch — APPROVED** [review]

**Comment by bryceml** [comment]
re-run full centos-7
re-run full windows
re-run full pr-ubuntu-2004-amd64-py3-pycryptodome
re-run full debian-11
re-run full arch
re-run full mojave

**Pull request merged**